rewrite-maven/src/test/java/org/openrewrite/maven/utilities/PrintMavenAsCycloneDxBomTest.java

```diff
@@ -102,4 +102,70 @@ void cycloneDxBom() {
         );
 
     }
+
+    @Test
+    void pomPackaging_cycloneDxBom() {
+        Xml.Document pom = MavenParser.builder()
+          .build()
+          .parse(
+            """
+            <?xml version="1.0" encoding="UTF-8"?>
+              <project xmlns="http://maven.apache.org/POM/4.0.0"
+                       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                       xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+                <modelVersion>4.0.0</modelVersion>
+              
+                <groupId>org.example</groupId>
+                <artifactId>pom_packaging</artifactId>
+                <version>1.0</version>
+                <packaging>pom</packaging>
+              
+                <properties>
+                  <maven.compiler.source>11</maven.compiler.source>
+                  <maven.compiler.target>11</maven.compiler.target>
+                </properties>
+              
+              
+                <dependencies>
+                  <dependency>
+                    <groupId>org.junit.jupiter</groupId>
+                    <artifactId>junit-jupiter-api</artifactId>
+                    <version>5.9.3</version>
+                    <scope>test</scope>
+                  </dependency>
+                </dependencies>
+              </project>
+            """
+          ).findFirst().orElseThrow(() -> new IllegalArgumentException("Could not parse as XML"));
+
+        String bom = PrintMavenAsCycloneDxBom.print(pom)
+          .replaceAll("<timestamp>.*</timestamp>", "<timestamp>TODAY</timestamp>");
+
+        String expectedQualifier = "type=pom";
```

Comment:
```suggestion
```

Reply:
I'd lean towards hardcoding the expectedQualifier in the expected value, as the value is not dynamic, and also present on the line `<component bom-ref="pkg:maven/org.example/pom_packaging@1.0?type=pom" type="library">`(?).
Would you agree with that change?

Reply:
I am okay with your suggestion as well. My intention was to make it clearer as otherwise it would be hard to find what should one focus on in the SBOM.